Makefile.common

```diff
@@ -21,9 +21,10 @@ ifeq ($(SYSTEM), win32)
   endif
   CSMLCSCODE = $(shell cygpath -w $(CSMLDIR)/csml.cs)
   EXECPREFIX=
-
+  PREFIX_DLL=
 else
-  CSC=gmcs
+  CSC=mcs
   CSMLCSCODE = $(CSMLDIR)/csml.cs
   EXECPREFIX=LD_LIBRARY_PATH=.
+  PREFIX_DLL=lib
```

Comment:
Can you explain the rationalr for the PREFIX_DLL under Unix?  Is it only to follow some convention, or does it address some issue?  Any reason not to do the same under Windows (if only to simplify the doc)?